src/docs/development/ios-14.md

```diff
@@ -1,13 +1,11 @@
 ---
-title: Developing for iOS 14 beta
-description: Describes a couple gotchas (and how to work around them) when developing for iOS14 on Flutter 1.20.
+title: Developing for iOS 14
+description: Describes a couple gotchas (and how to work around them) when developing for iOS 14.
 ---
 
 The iOS 14 release, the new version of Apple's mobile operating system,
-is coming. As it's not yet stable, we don't fully support it yet.
-
-But we have some tips to get you up and running&mdash;assuming
-that you don't mind being on the leading edge.
+is here. This page describes some known issues when developing for
+iOS 14.
 
 ## Using hot reload
```

Comment:
Sorry I meant this header. We can remove this now since it's included below.

Reply:
Oh! You want the other info back?

Reply:
Where's your doc again?